.eslintignore

```diff
@@ -3,4 +3,6 @@ storybook-static/**/*.js
 
 tsup.config.ts
 playwright.config.ts
-__playwright-tests__
\ No newline at end of file
+__tests__
```

Comment:
If this didn't need to change with the directory rename, maybe we can remove it?

Reply:
Oops, missed this one during the rename.